internal/controller/csiaddons/persistentvolumeclaim_controller.go

```diff
@@ -717,6 +723,62 @@ func (r *PersistentVolumeClaimReconciler) findChildEncryptionKeyRotationCronJob(
 	return activeJob, nil
 }
 
+// checkDisabledAnnotation checks if the annotation is set in the
+// PVC, namespace or the storage class. It returns true if the
+// annotation is set to `false`.
```

Comment:
Quite a confusing description if someone does not know the disabled annotation ends with `/enabled`. Please read it back to yourself, while you forgot the actual annotation key.